mitodl/bootcamp-ecommerce
# Sync new SMApply users with local User and Profile models

#### What are the relevant tickets?
closes #289 

#### What's this PR do?
This PR adds a periodic task that syncs SMApply users with the local User and Profile models. User JSON data is stored in a JSONField on Profile.

#### How should this be manually tested?
Open the terminal and run `sync_all_users()` from `smapply/tasks.py`
Check the User and Profile models to see that data was populated.
